e2e/tests/components/date-time-picker/date-time-picker.e2e-spec.ts

```diff
@@ -200,4 +200,13 @@ describe('Date Time Picker Tests', () => {
         }
 
     });
+
+    it('should not display horizontal line when select adjacent month', async () => {
+
+        let date = await page.getDate('31');
+        await date.click();
+
+        await page.nextDate.click();
```

Comment:
Here you should assert that the selected class is applied to the button of the 31st in the adjacent month.